### PR TITLE
test: add coverage for dataproxy construction inputs

### PR DESCRIPTION
**Pull request by jsstevenson**
* check invalid inputs
* ensure env var works

### CODE REVIEW

File: tests/extras/test_dataproxy.py [hunk 1]
```diff
@@ -20,3 +24,24 @@ def test_data_proxies(dp, request):
         "ga4gh:SQ.v_QTc1p-MUYdgrRv4LMT6ByXIOsdw3C_", start=0, end=50
     )
     assert seq == "CCTCGCCTCCGTTACAACGGCCTACGGTGCTGGAGGATCCTTCTGCGCAC"
+
+
+def test_invalid_data_proxy_uri():
```
Comment (korikuzma) [inline tests/extras/test_dataproxy.py:29]: Do we need to test if uri is none?

**Review by korikuzma — APPROVED** [review]
